include/rnp/rnp.h

```diff
@@ -1948,6 +1948,17 @@ RNP_API rnp_result_t rnp_key_get_default_key(rnp_key_handle_t  primary_key,
  */
 RNP_API rnp_result_t rnp_key_get_alg(rnp_key_handle_t key, char **alg);
 
+/** Get a SPHINCS+ key's parameter string
```

Comment:
Please put this under the `#if defined(RNP_EXPERIMENTAL_PQC)` like it is done for `rnp_op_generate_set_sphincsplus_param`